task17.js

```diff
@@ -0,0 +1,17 @@
+// Write a function that filters an array, leaving only elements of a certain type.
+
```

Comment:
it works OK, but i would recommend you to use your function from task 1(after you adding all updates for that task)